src/crypto-api.ts

```diff
@@ -39,14 +39,19 @@ export interface CryptoApi {
     globalBlacklistUnverifiedDevices: boolean;
 
     /**
-     * Checks if the user has previously published cross-signing keys
+     * Check if the cross signing keys for the user are available.
      *
-     * This means downloading the devicelist for the user and checking if the list includes
-     * the cross-signing pseudo-device.
+     * - If the user is tracked, a `/keys/query` request is made to update locally the cross signing keys.
+     * - If the user is not tracked locally and downloadUncached is set at true,
+     *   a `/keys/query` request is made to the server to retrieve the cross signing keys.
+     * - Otherwise, return false
      *
-     * @returns true if the user has previously published cross-signing keys
+     * @param userId - the user ID to check
+     * @param downloadUncached - If true, download the cross signing keys.
```

Comment:
Let's use the same wording as `getUserDeviceInfo` here.

```suggestion
     * @param downloadUncached - If true, download the device list for users whose device list we are not
     *    currently tracking. Defaults to false, in which case `false` will be returned for such users.
```